src/core/layout/qgslayoutitemattributetable.cpp

```diff
@@ -579,6 +581,26 @@ bool QgsLayoutItemAttributeTable::getTableContents( QgsLayoutTableContents &cont
           style = QgsConditionalStyle::compressStyles( styles );
         }
 
+        const QgsEditorWidgetSetup setup = layer->fields().at( idx ).editorWidgetSetup();
+
+        if ( ! setup.isNull() )
+        {
+          QgsFieldFormatter *fieldFormatter = QgsApplication::fieldFormatterRegistry()->fieldFormatter( setup.type() );
+          QVariant cache;
+
+          if ( mLayerCache.contains( column.attribute() ) )
+          {
+            cache = mLayerCache[ column.attribute() ];
```

Comment:
minor thing, but instead of the calls to both contains and [] it'd be more efficient to go:
   

    auto it = mLayerCache.constFind( column.attribute() );
    if ( it ! = mLayerCache.constEnd() )
        cache = it.value();

Reply:
Done, thanks!